game_cli.rb

```diff
@@ -18,7 +18,7 @@ def run
         while !@new_game_driver.await.turn_over?.value
           print_permanents(activePlayer, prompt="here are the permanents you have:")
 
-          cardToPlay = @interface.await.choose_from_list(hand, "Select a card from your hand to play").value
+          cardToPlay = @interface.await.choose_from_list(hand, :select_a_card_to_play_prompt).value
```

Comment:
Definitely seems like a smell that there is any prompting going on this this class at all. Something to consider looking into